StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1608ElementDocumentationMustNotHaveDefaultSummary.cs

```diff
@@ -50,7 +55,35 @@ public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics
         /// <inheritdoc/>
         public override void Initialize(AnalysisContext context)
         {
-            // TODO: Implement analysis
+            context.RegisterSyntaxNodeAction(HandleDocumentation, SyntaxKind.SingleLineDocumentationCommentTrivia, SyntaxKind.MultiLineDocumentationCommentTrivia);
+        }
+
+        private void HandleDocumentation(SyntaxNodeAnalysisContext context)
+        {
+            var documentationTrivia = context.Node as DocumentationCommentTriviaSyntax;
+
+            if (documentationTrivia != null)
+            {
+                var summaryElement = XmlCommentHelper.GetTopLevelElement(documentationTrivia, XmlCommentHelper.SummaryXmlTag) as XmlElementSyntax;
+
+                if (summaryElement != null)
+                {
+                    var textElement = summaryElement.Content.FirstOrDefault() as XmlTextSyntax;
+
+                    if (textElement != null)
+                    {
+                        string text = XmlCommentHelper.GetText(textElement);
```

Comment:
You should call `GetText(textElement, true)` here. Setting `normalizeWhitespace` to true covers cases where there are multiple spaces and/or newline characters words.